test/data/remotes.js

```diff
@@ -10,5 +10,13 @@ export default {
   bitbucket: {
     hostname: 'bitbucket.org',
     url: 'https://bitbucket.org/user/repo'
+  },
+  azure: {
+    hostname: 'dev.azure.com',
+    url: 'https://dev.azure.com/organization/project/_git/repo'
+  },
+  visualstudio: {
+    hostname: 'organization.visualstudio.com',
+    url: 'https://organization.visualstudio.com/project/_git/repo'
```

Comment:
We should add `project` to these (or `issueUrl` if we rename it)?

Reply:
I don't know if is 100% necesary add 'project' url there, since it is the same as 'url' 
but without '/_git/repo'

Reply:
Added in ffd9d3cac778694be237385c6effcf9540aba022, also I changed `organization` by `user`